config/setup_logging.rb

```diff
@@ -0,0 +1,135 @@
+# Logging config in one place instead of six
+
+class SetupLogging
+  attr_accessor :config
+
+  def initialize(config)
+    self.config = config
+  end
+
+  def run!
+    _configure_lograge
+
+    if Rails.env.development? || ENV['TEST_DEVELOPMENT_LOGGING_CONFIG'] == 'true'
```

Comment:
I went for copying  from the warehouse over trying to exactly match the pre-existing logging config.